doc/JSON-RPC-interface.md

```diff
@@ -5,6 +5,36 @@ The headless daemon `bitcoind` has the JSON-RPC API enabled by default, the GUI
 option. In the GUI it is possible to execute RPC methods in the Debug Console
 Dialog.
 
+## Endpoints
+
+There are two JSON-RPC endpoints on the server:
+
+1. `/`
+2. `/wallet/<walletname>/`
+
+### `/` endpoint
+
+This endpoint is always active and can service both wallet and non-wallet
+requests when there are zero or one wallets loaded.
```

Comment:
I see your point here, but I was also trying to distinguish: active=enabled, services=wallet/non-wallet.

How about:

> This endpoint is always active. It can always service non-wallet requests and can service wallet requests when exactly one wallet is loaded.

Reply:
Done